WalletWasabi/CoinJoin/Client/Clients/CoinJoinClient.cs

```diff
@@ -191,7 +191,7 @@ public void Start()
 			finally
 			{
 				Interlocked.CompareExchange(ref _running, StateStopped, StateStopping); // If IsStopping, make it stopped.
-				}
```

Comment:
LOC250 there is another warning. Could you please fix that also?

```diff
-StateUpdated?.Ivoke(this, null);
+StateUpdated?.Invoke(this, EventArgs.Empty);
```

Is no one subscribed to the `StateUpdated` event? Weird.

Reply:
As far as I know, itt will be replaced later anyway.